tests/phpunit/includes/abstract-testcase.php

```diff
@@ -1070,6 +1070,41 @@ public function assertNonEmptyMultidimensionalArray( $actual, $message = '' ) {
 		}
 	}
 
+	/**
+	 * Assert that two text strings representing file paths are the same, while ignoring
+	 * OS-specific differences in the directory separators.
+	 *
+	 * This allows for tests to be compatible for running on both *nix based as well as Windows OS.
+	 *
+	 * @since 6.7.0
+	 *
+	 * @param string $path_a File or directory path.
+	 * @param string $path_b File or directory path.
+	 */
+	public function assertSamePathIgnoringDirectorySeparators( $path_a, $path_b ) {
+		$path_a = $this->normalizeDirectorySeparatorsInPath( $path_a );
+		$path_b = $this->normalizeDirectorySeparatorsInPath( $path_b );
+
+		$this->assertSame( $path_a, $path_b );
+	}
+
+	/**
+	 * Normalize directory separators in a file path to be a forward slash.
+	 *
+	 * @since 6.7.0
+	 *
+	 * @param string $path File or directory path.
+	 *
+	 * @return string
+	 */
+	public function normalizeDirectorySeparatorsInPath( $path ) {
+		if ( ! is_string( $path ) || PHP_OS_FAMILY !== 'Windows' ) {
+			return $path;
+		}
+
+		return \strtr( $path, '\\', '/' );
```

Comment:
tiny nit for consistency:
```suggestion
		return strtr( $path, '\\', '/' );
```

Reply:
Thanks for catching that. I'm getting too used to codebases where that's the norm ;-)

Reply:
I completely get it, especially knowing how often you switch between codebases :)